README.md

```diff
@@ -34,42 +34,21 @@ curl -O https://zowe.jfrog.io/zowe/libs-snapshot-local/org/zowe/sample-trial-app
 # From local - if downloaded on z/OS skip this
 sftp ibmuser@mymainframe.ibm.com
 put <pax-name>.pax
-
-# On z/OS
-mkdir sample-node-api
-cd sample-node-api
-pax -ppx -rf ../<pax-name>.pax
 ```
 
 
 ## PART II: Deploy with Zowe on server
 
-## Method 1: Manually
-1) `ssh ibmuser@my.mainframe.com`
-
-2) Go to your sample trial app folder
-    Install as zowe desktop app   
-    ```
-    cd ~/zowe/instance/bin
-    install-app.sh </usr/lpp/extender>/sample-trial-app/
-    ```
-    
-3) This app uses dataservice api as which can be deployed and started separately:
-`https://github.com/zowe/sample-node-api`
-
-## Method 2: Using Zowe Lifecycle scripts
-### 1) Register as EXTERNAL_COMPONENTS with zowe
-
-Use property `EXTERNAL_COMPONENTS` located in file `$INSTANCE_DIR/instance.env`       
-Append it (comma separated) with the directory containing your service lifecycle scripts.
+### 1) login
+```  
+ssh ibmuser@my.mainframe.com       
+```
 
-In our sample it is:   
+### 2) install component using zowe-install-component.sh script
 ```
- vi INSTANCE_DIR/instance.env   
- EXTERNAL_COMPONENTS=</usr/lpp/extender>/sample-node-api/bin,</usr/lpp/extender>/sample-trial-app/bin      
+./<zowe-runtime-dir>/bin/zowe-install-component.sh -c <component-name> -i <zowe-instance-dir> -o <component-pax-file> -l <log-folder>

```

Comment:
-c here is not important because our package has manifest, so the installer can get this info.
-d is more worthy to mention let's say `-d <zowe-extensions-dir>`